puts/harness/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(puts VERSION 0.1 LANGUAGES C)
+
+enable_testing()
```

Comment:
Do we use this?

Reply:
Not sure what you are talking about, so I'll answer for all lines 😄

- We set a consistent minimal cmake version because it provides a common set of policies on how cmake works and which functions are available.
- We need the C language to get the C toolchain to build the harness and bundle the PUT into a single object file. This allows us to make sure all symbols coming from the vendor library (e.g. OpenSSL) are local, which allows us in turn to link statically several PUTs including the same vendor library into tlspuffin (e.g. OpenSSL 1.1.1k + OpenSSL 1.1.1j)
- We enable testing as it is generally a good practice in cmake builds. It gives access to the `test` target (empty in that case) and allows users to do the generic `make && make test && make install` (or equivalent for other back-ends) without knowing anything about your project.